### DISCUSSION TITLE
FE 요구사항

### DISCUSSION THREAD

**Discussion by EeeasyCode** [TEAM-YOAJUNG/backend-server#36] [General]
- [ ] request, response swagger 필드에 type이나 optional인지 등에 대한 확실한 명시 필요
- [ ] Club ID -> UUID가 아닌, Auto Increament로 수정

**Comment by EeeasyCode** [comment]
@kangju2000 여기에 요청해주시면 됩니다~ 따로 연락주시면 여기에 제가 추가해놓는 것도 괜찮습니다!

**Comment by kangju2000** [comment]
동아리목록 조회에서 `clubId`도 리턴 부탁드립니당

**Reply by EeeasyCode** [reply]
넵~

**Comment by kangju2000** [comment]
login 시 `Authorization 헤더가 없습니다.`라는 메시지로 401 에러가 뜹니다. login할 때는 권한이 필요없지 않을까요??

**Reply by EeeasyCode** [reply]
이 부분은 수정했어요!

**Comment by kangju2000** [comment]
refresh token 필요

**Reply by kangju2000** [reply]
verify accessToken

성공 -> refreshToken
실패 -> 401

**Comment by kangju2000** [comment]
@opp-13 

`GET:/api/v1/application/list`

- application 도메인 swagger의 **required 처리** 필요
- `ToGetApplicationHistoryDTO`에 **지원서 제출일** 필요 
<img width="881" alt="image" src="https://github.com/user-attachments/assets/037fd300-9efa-473f-aaf5-b3b5b568a772">


```ts
// generated in Swagger OpenAPI
export interface ResFormToGetApplicationHistoryListDTO {
  /** @format date-time */
  time?: string;
  code?: string;
  message?: string;
  result?: ToGetApplicationHistoryListDTO;
}

export interface ToGetApplicationHistoryListDTO {
  toGetApplicationHistoryDTO?: ToGetApplicationHistoryDTO[];
}

export interface ToGetApplicationHistoryDTO {
  /** @format int64 */
  applicationId?: number;
  clubName?: string;
  status?: string;
}
```

**Reply by opp-13** [reply]
수정 완료했습니다.

**Comment by kangju2000** [comment]
@EeeasyCode 

- `results`도 swagger에서 required 처리 필요

```ts
export interface ArrayResponseClubSummaryResponse {
  results?: ClubSummaryResponse[];
}
```

**Reply by EeeasyCode** [reply]
처리 완료

**Comment by kangju2000** [comment]
@jungyuminn 
회원가입 페이지에서 이메일 인증할 때, 가입한 유저면 이미 가입된 유저라고 알려주는게 좋을 것 같은데 어떻게 생각하시나요?

현재는 최종적으로 회원가입 요청을 날릴 때 가입 판단 여부 로직을 거치고 있네용

**Reply by jungyuminn** [reply]
인증코드 발송 버튼 누른 시점에서 처리되길 원하시는건가요?

**Comment by kangju2000** [comment]
@EeeasyCode recruitment 리스트 요청 시 club imageurl도 보내주기
